server/tests-py/test_events.py

```diff
@@ -81,6 +84,34 @@ def test_create_operation_spec_not_provider_err(self, hge_ctx):
     def dir(cls):
         return 'queries/event_triggers/create-delete'
 
+# Smoke test for handling a backlog of events
```

Comment:
Seems like we are testing if we process 2000 events. But how do we know that backpressure changes are working i.e. we are emitting warnings?

Reply:
We don't, this is just a smoke test for code coverage and so I could convince myself it was working. Do you think it's worth the effort to test for logging?

Reply:
Okay, just wanted to clarify. I think checking for log line might be too much infra. Is there any other way to detect if things are actually getting throttled?

Reply:
I can't think of a slick way to do it... maybe better to punt on that?